### PR TITLE
Comment overflow fixed.

### PR DESCRIPTION
**Pull request by krystianbajno**
CSS rule added, fixed the overflow on long words.

### CODE REVIEW

**Comment by krystianbajno** [comment]
Image
https://ibb.co/bW5ysH

**Comment by krystianbajno** [comment]
Img
https://ibb.co/kKjGCH
https://ibb.co/iNDXmc

File: assets/style/commento.css [hunk 1]
```diff
@@ -80,6 +80,7 @@
     margin-top: 0px;
     margin-bottom: 8px;
     border-top: 1px solid #f0f0f0;
+    word-wrap:break-word;
```
Comment (adtac) [inline assets/style/commento.css]: space after colon please :)

**Comment by krystianbajno** [comment]
Okay, I added the space and changed commit name. @adtac

**Comment by adtac** [comment]
https://github.com/adtac/commento/pull/173/commits/80d44868f3552661485761714d726d6a17cf5183

**Comment by adtac** [comment]
ack https://github.com/adtac/commento/pull/173/commits/80d44868f3552661485761714d726d6a17cf5183

**Pull request closed without merging**